app/javascript/components/mentoring/session/PreviousSessionsLink.tsx

```diff
@@ -12,7 +12,7 @@ export const PreviousSessionsLink = ({
 
   return (
     <div className="previous-sessions">
-      {numSessions} previous sessions
+      See {numSessions} previous discussions
```

Comment:
Do we need to singularize/plurarize the `"discussions"` word?

Reply:
Good spot. Yes.